Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Views/Navigation/SyncStatusData.swift

```diff
@@ -6,37 +6,16 @@ struct SyncStatusData {
     let networkId: String
     let isHidden: Bool
     
-    var title: String {
-        guard networkId.isNotEmpty else { return Loc.SyncStatus.LocalOnly.title }
-        switch status {
-        case .unknown:
-            return Loc.preparing
-        case .offline:
-            return Loc.noConnection
-        case .syncing:
-            return Loc.syncing
-        case .synced:
-            return Loc.synced
-        case .failed, .incompatibleVersion, .UNRECOGNIZED:
-            return Loc.notSyncing
-        }
-    }
+    var isLocalOnly: Bool { networkId.isEmpty }
+    var isNotLocalOnly: Bool { !isLocalOnly }
     
-    var description: String {
-        guard networkId.isNotEmpty else { return Loc.SyncStatus.LocalOnly.description }
+    var title: String {
+        guard isNotLocalOnly else { return "" }
         switch status {
-        case .unknown:
-            return Loc.initializingSync
-        case .offline:
-            return Loc.nodeIsNotConnected
-        case .syncing:
-            return Loc.downloadingOrUploadingDataToSomeNode
-        case .synced:
-            return syncedDescription
-        case .failed:
-            return Loc.failedToSyncTryingAgain
+        case .unknown, .offline, .syncing, .synced, .failed:
+            return ""
         case .incompatibleVersion, .UNRECOGNIZED:
-            return Loc.Sync.Status.Version.Outdated.description
+            return Loc.incompatibleVersion
         }
     }
```

Comment:
if we remove description for now, maybe we should remove strings?

Reply:
I am saving them for now. There will be a new info menu, and they will be reused there.